src/sentry/migrations/0536_backfill_tombstones.py

```diff
@@ -52,8 +26,8 @@ class Migration(CheckedMigration):
 
     operations = [
         migrations.RunPython(
-            backfill_tombstones,
+            migrations.RunPython.noop(),
```

Comment:
```suggestion
            migrations.RunPython.noop,
```

Should this pass the function instead of its result?